src/mapper/mapAnyToOrder.ts

```diff
@@ -1,14 +1,19 @@
-import { Order } from '@airswap/typescript';
+import { DbOrder } from 'model/DbOrder.js';
+import { toNumber } from '../converter/index.js';
+
+export function mapAnyToDbOrder(data: any): DbOrder {
+    if (!data || typeof data != 'object') {
+        return undefined;
+    }
 
-export function mapAnyToOrder(data: any): Order {
     return {
```

Comment:
All values should be strings because JavaScript has problems handling big numbers. Either the database should parse into a number internally or BigNumber should be used here.

Reply:
Fixed !
All strings for ui/smart contract, I added an approximative field in order to sort and limit